### PR TITLE
Add contributor page and fix some minor issues

### PR DESCRIPTION
**Pull request by SubhamB2003**
issue solved: #648 

https://github.com/narayan954/dummygram/assets/94003231/77f1da07-5ceb-4979-ac15-d128b9da7b2d

I also fix some minor issues in the setting the About page and privacy and policy page link are incorrect. I also fixed this

### CODE REVIEW

**Comment by narayan954** [comment]
Thanks @SubhamB2003 reviewing

**Comment by narayan954** [comment]
Also, as a tip... @SubhamB2003 if you write something like `solves #648` instead of `issue solved: #648 ` in the pull request comment, you can directly link your issue to the pull request :)

File: src/pages/FooterPages/ContributorPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+import ContributorCard from "./ContributorCard";
+
+function Contributor() {
+  const [contributors, setContributors] = useState([]);
+  const isNonMobileScreen = useMediaQuery("(max-width: 800px)");
+
+  const getData = async () => {
+    const res = await fetch(
+      "https://api.github.com/repos/narayan954/dummygram/contributors?per_page=100"
```
Comment (narayan954) [inline src/pages/FooterPages/ContributorPage/index.jsx:16]: you're limiting this list to 100 contributors only, which works for now, but not in possible future.Need to fix this.

File: src/pages/FooterPages/ContributorPage/index.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  min-height: 100vh;
+  border-radius: 60px;
+  padding-bottom: 40px;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: center;
+}
+
+.grad5 {
```
Comment (narayan954) [inline src/pages/FooterPages/ContributorPage/index.css]: aren't these supposed to be in design.css ?

File: src/pages/FooterPages/ContributorPage/index.jsx (hunk 1 above)
Comment (narayan954) [reply src/pages/FooterPages/ContributorPage/index.jsx:16]: Also, as a next step, we can add pagination to this

**Review by narayan954 — APPROVED** [review]
some changes have been made by me, rest we can do later. For now, do resolve the conversations

File: src/pages/FooterPages/ContributorPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <div className="contributor-container footer-page-para-color">
+      <div className="contributor-sub-container">
+        <span className="grad1 grad"></span>
```
Comment (narayan954) [inline src/pages/FooterPages/ContributorPage/index.jsx:37]: also, can you remind me, what're these spans for? the jelly background effect?

File: src/components/SettingsComponents/Sidebar/index.jsx [hunk 4]
```diff
@@ -21,10 +21,7 @@ const SettingsSidebar = () => {
           >
             Notification
           </Link>
-          <Link
-            className="settings-sidebar-item"
-            to="/dummygram/settings/about"
-          >
+          <Link className="settings-sidebar-item" to="/dummygram/about">
```
[the remark this answers is not in the document]
Comment (SubhamB2003) [reply src/components/SettingsComponents/Sidebar/index.jsx]: oh ok

File: src/pages/FooterPages/ContributorPage/index.css (hunk 2 above)
Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.css]: Yes, but I use this only on the contributor page so I write this code in this CSS file.

File: src/pages/FooterPages/ContributorPage/index.jsx (hunk 3 above)
Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.jsx:37]: > also, can you remind me, what're these spans for? the jelly background effect?
Yes

Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.jsx:37]: yes

File: src/pages/FooterPages/ContributorPage/index.jsx (hunk 1 above)
Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.jsx:16]: Hey, can I try to add the pagination?

Comment (narayan954) [reply src/pages/FooterPages/ContributorPage/index.jsx:16]: yes, create separate issue :)

File: src/pages/FooterPages/ContributorPage/index.css (hunk 2 above)
Comment (narayan954) [reply src/pages/FooterPages/ContributorPage/index.css]: if we add them in design.css we can use them anywhere?

Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.css]: yes

File: src/pages/FooterPages/ContributorPage/index.jsx (hunk 1 above)
Comment (SubhamB2003) [reply src/pages/FooterPages/ContributorPage/index.jsx:16]: ok thanks

**Pull request merged**